server/mdm/mdm.go

```diff
@@ -108,3 +108,9 @@ func FleetReservedProfileNames() map[string]struct{} {
 func ListFleetReservedWindowsProfileNames() []string {
 	return []string{FleetWindowsOSUpdatesProfileName}
 }
+
+// ListFleetReservedMacOSProfileNames returns a list of PayloadDisplayName strings
+// that are reserved by Fleet for macOS.
+func ListFleetReservedMacOSProfileNames() []string {
```

Comment:
nit: is this only used for tests?

Reply:
Yes, currently, but I believe it will be used in the datastore with upcoming DDM OS updates